DIMS-Core/DIMS-Core.DataAccessLayer/Repositories/VUserTaskRepository.cs

```diff
@@ -0,0 +1,31 @@
+using DIMS_Core.DataAccessLayer.Interfaces;
+using DIMS_Core.DataAccessLayer.Models;
+using Microsoft.EntityFrameworkCore;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace DIMS_Core.DataAccessLayer.Repositories
+{
+    public class VUserTaskRepository : IReadOnlyRepository<VUserTask>
```

Comment:
Did you implement readonly repository?

Reply:
No, because I saw implementation of VUserProfileRepository class in main branch (VUserProfileRepository.cs file) and I implemented this class with an example of that class

Reply:
I inplemented ReadOnlyRepository class